ground-station-openmct/terminal.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <style>
+      body {
+        background-color: #000;
+        color: #fff;
+        font-family: "Courier New", monospace;
+        margin: 0;
+        padding: 0;
+        overflow: hidden;
+      }
+      #terminal {
+        width: 80%;
+        max-width: 800px;
+        height: 400px;
+        margin: 20px auto;
+        background-color: #000;
+        border: 1px solid #fff;
+        overflow-y: scroll;
+        white-space: pre-wrap;
+      }
+      #terminal::-webkit-scrollbar {
+        display: none;
+      }
+      #input {
+        width: 100%;
+        background: #000;
+        border: none;
+        color: #fff;
+        outline: none;
+        caret-color: #fff;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="terminal"></div>
+    <input id="input" placeholder="$" autocomplete="off" />
+    <script>
```

Comment:
Put the JavaScript code in a different file, and then call the js code in the script tag.

Ex: <script src="terminal-ui.js"></script>